client/signup/steps/design-picker/index.jsx

```diff
@@ -51,12 +42,10 @@ export default function DesignPickerStep( props ) {
 		siteHasFeature( state, siteId, WPCOM_FEATURES_PREMIUM_THEMES )
 	);
 
-	const userLoggedIn = useSelector( ( state ) => isUserLoggedIn( state ) );
-
 	const dispatch = useDispatch();
 	const translate = useTranslate();
 
-	const [ selectedDesign, setSelectedDesign ] = useState( null );
+	const [ selectedDesign ] = useState( null );
```

Comment:
Could the `selectedDesign` variable be eliminated completely? It's now always `null`.

Reply:
Nice catch, done, thanks!